### PR TITLE
Update fio to 2.3 version and add fio job file for ppc platform

### PR DESCRIPTION
**Pull request by maxujun**
Update fio to 2.3 version because 2.1 version can't complile successfully on ppc platform
Add fio job file for pcc platform because the mmap test need block size no less than pagesize 64k

ID:1277044

### CODE REVIEW

File: fio/fio.py [hunk 1]
```diff
@@ -39,9 +39,14 @@ def run_once(self, opts=None, job=None, user='root'):
 
         if opts:
             _opts += opts
+        arch_output = commands.getoutput("arch")
```
Comment (xutian) [inline fio/fio.py]: we can use platform module to get host arch.
code like:
import platform
arch = platform.machine()
job_file = "fio-mixed.job"
if "ppc64" in arch:
     job_file = 'fio-mixed-ppc.job'

Comment (maxujun) [inline fio/fio.py]: yes,I knew that module after pull request.need i modify it ?

Comment (xutian) [inline fio/fio.py]: sure, please update your PR.

Comment (maxujun) [inline fio/fio.py]: ok

**Comment by xutian** [comment]
looks good to me. merged thanks @maxujun

**Pull request merged**